spec/factories/dishes.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :dish do
+    name "MyString"
+    description "MyString"
+    price 10
+    menu
```

Comment:
I would like you to make sure that the dish you create with this factory actually belongs to a menu.